recipes/verilog-slang/all/conanfile.py

```diff
@@ -0,0 +1,96 @@
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get, rm, rmdir, replace_in_file
+import os
+
+
+required_conan_version = ">=2.0.9"
+
+
+class VerilogSlangConan(ConanFile):
+    name = "verilog-slang"
+    description = "SystemVerilog compiler and language services"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://sv-lang.com/"
+    topics = ("parse", "compiler", "verilog", "language-service", "systemverilog")
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+    implements = ["auto_shared_fpic"]
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def validate(self):
+        check_min_cppstd(self, 20)
+
+    def build_requirements(self):
+        self.tool_requires("cmake/[>=3.20 <5]")
+
+    def requirements(self):
+        self.requires("fmt/12.1.0")
+        if not self.options.shared or not self.settings.os == "Windows":
+            self.requires("mimalloc/2.2.4")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+        # INFO: Let Conan handle the C++ standard by settings.compiler.cppstd
+        replace_in_file(self, os.path.join(self.source_folder, "CMakeLists.txt"),
+                        "set(CMAKE_CXX_STANDARD",
+                        "# set(CMAKE_CXX_STANDARD")
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.cache_variables["SLANG_INCLUDE_TESTS"] = False
```

Comment:
Do we need the tools?

```suggestion
        tc.cache_variables["SLANG_INCLUDE_TESTS"] = False
        tc.cache_variables["SLANG_INCLUDE_TOOLS"] = False
```